powertools-idempotency/powertools-idempotency-core/src/main/java/software/amazon/lambda/powertools/idempotency/internal/IdempotencyHandler.java

```diff
@@ -37,6 +32,12 @@
 import software.amazon.lambda.powertools.idempotency.persistence.DataRecord;
 import software.amazon.lambda.powertools.utilities.JsonConfig;
 
+import java.time.Instant;
```

Comment:
I think you don't have [checkstyle](https://github.com/aws-powertools/powertools-lambda-java/blob/main/CONTRIBUTING.md#dev-setup) applied. Can you configure intelliJ and make sure to reformat files properly?